broad/broad/spiders/broad.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import re
+import datetime
+
+from six.moves.urllib.parse import urlparse
+import click
+import scrapy
+from scrapy.http import Request, HtmlResponse
+from scrapy.linkextractors import LinkExtractor
+
+from broad.items import Page
+
+
+class BroadSpider(scrapy.Spider):
+    name = "broad"
+    ratings_map = {
+        'one': 1,
+        'two': 2,
+        'three': 3,
+        'four': 4,
+        'five': 5,
+    }
+
+    with open('domains.txt') as f:
+        start_urls = [url.strip() for url in f.readlines()]
```

Comment:
I would prefer for this urls to be generated dynamically, not read from the file, because this way it would be easier to change number of domains, port, host, etc.